esp-hal/src/gpio/mod.rs

```diff
@@ -46,22 +44,7 @@
 //! an input and output signal. You can then pass these signals to the
 //! peripheral drivers similar to how you would pass a pin.
 //!
-//! ## Examples
```

Comment:
Why did you drop the Examples header?

Reply:
I don't think the module level needs a general set of examples. I think that if a section needs them (like interconnect), that section can list examples.